### DISCUSSION TITLE
Improving Participatory Mapping and Critical Cartography

### DISCUSSION THREAD

**Discussion by okcirE** [albertkun/21S-ASIAAM-191A#78] [Past Thinking Caps]
We can better utilize participatory mapping and critical cartography for civil society through education and awareness of the processes of defining and constructing maps. I was never exposed to the potential harm that some maps could infer in a given population, until this course. Therefore, I do believe courses even in early education (high school elective, maybe?) on topics that focus on disintegrating and analyzing map data are critical to achieving better utilization of these methods. Furthermore, I am not sure if this has yet to exist yet, but the implementation of a map-accountability program could prove effective. This program would serve as fact-checkers as well as a guide for map creators to be able to discern whether their map can potentially harm after dissemination. Definitely, I think there is much work still to be done in this area of expertise and I am glad that this course are increasing awareness for students, like me.

**Comment by alizezila** [comment]
Thanks for sharing! I think your point about map accountability is very important. As map and GIS professionals, we must be cognizant of the messages we are sending, the stories we are telling. I agree that a program that can fact-check data and potentially prevent harm from these maps is a great idea, though it is important that real people who are part of the communities that larger-scale maps might be representing have a say in the process, not just rich white educated people who may or may not have better access to those positions of power.
